https://issues.redhat.com/browse/ACM-15267 (#7241)
open-cluster-management/rhacm-docs: networking/submariner/subm_configure.adoc

* For {ocp-short} ARM deployments, you must use the `c6g.large` `instanceType` or any other available instance type.
* If the gateway nodes are directly reachable over their private IPs without any NAT in between, make sure that the firewall configuration allows the ESP protocol on the gateway nodes.
+
*Notes:* 
+

> ** This is configured automatically when your clusters are deployed in an Amazon Web Services, Google Cloud Platform, Microsoft Azure, or Red Hat OpenStack environment, but must be configured manually for clusters on other environments and for the firewalls that protect private clouds.

Reviewer: maybe specify what "this" is and make the last sentence more user focused. 

What do you think of the following adjustments?:

```suggestion
** This update is configured automatically when your clusters are deployed in an Amazon Web Services, Google Cloud Platform, Microsoft Azure, or Red Hat OpenStack environment. You must manually configure it for clusters on other environments and for the firewalls that protect private clouds.
```
